AdyenDropIn/Utilities/ComponentManager.swift

```diff
@@ -30,14 +30,17 @@ internal final class ComponentManager {
     internal let order: PartialPaymentOrder?
     
     internal let apiContext: APIContext
+
+    internal let presentationDelegate: PresentationDelegate
```

Comment:
this should be weak, right? since the `presentationDelegate` is dropIn and dropIn owns the `Componentmanager`

Reply:
You're right! Fixed